app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/activeDirectoryAuthForm/activeDirectoryAuthFormFields/activeDirectoryAuthFormFields.jsx

```diff
@@ -5,15 +5,17 @@ import { FormField } from 'components/fields/formField';
 import { FieldErrorHint } from 'components/fields/fieldErrorHint';
 import { Input } from 'components/inputs/input';
 import { InputConditional } from 'components/inputs/inputConditional';
-import { LDAP_PREFIX, LDAPS_PREFIX } from '../../../../common/constants';
 import {
-  DOMAIN_KEY,
+  LDAP_ATTRIBUTES_KEY,
+  LDAP_PREFIX,
+  LDAPS_PREFIX,
   BASE_DN_KEY,
   EMAIL_KEY,
   FULL_NAME_KEY,
   URL_KEY,
   PHOTO_KEY,
-} from '../constants';
+} from '../../../../../common/constants';
```

Comment:
Better use absolute path. The relative one is too deep.

Reply:
Forget about some files, sorry for misleading.
Fixed